### DISCUSSION TITLE
如何在texshop中引用参考文献

### DISCUSSION THREAD

**Discussion by Vvxir** [TheNetAdmin/zjuthesis#174] [General]
您好，我想请问一下如何在mac的texshop中引用参考文献，我是生成了BibTex格式文件后复制到body/ref.bib上的，之后在原文中\cite{}但是出现的是我写的内容变成了文字上标，我尝试在package里面加上[backend=bibtex]但还是编译不出，textshop只能用xelatex和bibtex没有看到之前同学说的biber，求解决方法！还有请问参考文献会自动在附录前生成吗，如果不能是否需要加什么代码？拜托各位大佬了QAQ
<img width="229" alt="E2883D7A-E159-45D3-ABCC-525B8C85FE40" src="https://user-images.githubusercontent.com/68053100/118700905-b7c00100-b845-11eb-8588-408a3c76717a.png">

**Comment by TheNetAdmin** [comment]
没用过 texshop，改用 overleaf 或者vscode试试吧，这些有教程

**Comment by hushidong** [comment]
参考文献编译用命令行，不一定要用IDE的。 hzzmail ***@***.*** 签名由 网易邮箱大师 定制 在2021年05月19日 02:52，Zixuan Wang 写道： 没用过 texshop，改用 overleaf 或者vscode试试吧，这些有教程 — You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub, or unsubscribe.

**Reply by Vvxir** [reply]
我用mac的终端试了一下好像还是不行TAT

**Comment by zepinglee** [comment]
texshop 需要手动调用 biber 编译。另外不要用 `[backend=bibtex]`。

**Reply by Vvxir** [reply]
请问怎么调用，有具体一点方法吗？QAQ

**Comment by hushidong** [comment]
命令行或终端，到你文件所在目录下，输入命令biber zjuthesis，回车。就这么简单。 hzzmail ***@***.*** 签名由 网易邮箱大师 定制 在2021年05月19日 14:24，BeiniZhao 写道：

**Reply by Vvxir** [reply]
谢谢解答！但是好像还是没有解决，我也不知道什么问题害TAT，它就一直是我输入的文字作为上标，也没有参考文献页，尝试了overleaf没办法compile显示超时了

**Comment by hushidong** [comment]
你就先不要用ide了，先把pdf删掉，用终端进你的目录。三步编译 xelatex zjuthesis biber zjuthesis xelatex zjuthesis 如果还不行，就写个简单的文档，文档内使用biblatex，引用一篇文献，编译一下，看结果，看你的环境是不是ok的。 hzzmail ***@***.*** 签名由 网易邮箱大师 定制 在2021年05月19日 15:28，BeiniZhao 写道： 我用mac的终端试了一下好像还是不行TAT — You are receiving this because you commented. Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by hushidong** [comment]
你跑个简单示例看看，比如：
test.tex
```
\documentclass{article}
\usepackage[style=gb7714-2015,gbpunctin=false]{biblatex}
\begin{filecontents}{\jobname.bib}
@collection{somebook,
  editor =    {Edward Editor},
  title = {Some Book},
  year =      2020,
  publisher = {Polly Publisher},
}

@book{newbook,
  editor =    {Edward Editor},
  title = {New Book},
  year =      2020,
  publisher = {Polly Publisher},
}
\end{filecontents}
\addbibresource{\jobname.bib}

\begin{document}

\cite{somebook,newbook}
\printbibliography
\end{document} 
```

命令行编译：
 xelatex test
biber test
 xelatex test

看结果，是否正确：

正常是这样的：
![图片](https://user-images.githubusercontent.com/20421795/118839312-fa8ee100-b8f8-11eb-8bcd-b8614f2d113e.png)
